uiowa/uiowa
# Update drupal core dependencies to 9.5

Resolves #6061

# How to test

```
ddev blt ds --site sandbox.uiowa.edu && ddev drush @sandbox.local uli
```
- Spot check functionality to try to find regressions.
- Attempt to review the changes: https://git.drupalcode.org/project/drupal/-/compare/9.4.14...9.5.9

## LB off-canvas resizing regression
- https://sandbox.uiowa.ddev.site/node/1386/layout
- Click 'Add section' and note the width of the off-canvas tray
- Click 'Add block' and note that it displays in a wider tray
- Click 'Configure' on an existing block and note that it displays in a wider tray

## Validate patch changes
### Updating media attributes in layout builder blocks works

- https://sandbox.uiowa.ddev.site/node/1606/layout
- Add a text area block
- Add an image
- Save block
- Configure the block you just saved
- Change the image "Display" to another format and save the block
- Configure the block you just updated
- Confirm that your changes are saved

### Selected style options in list blocks are preserved when pager links are clicked

- https://sandbox.uiowa.ddev.site/debug-one-article-list-block or https://sandbox.uiowa.ddev.site/debug-one-people-list-block
- Click layout tab, configure the list block
- Check the checkbox to "Show pager" and save the block
- Click through pager links to ensure that displayed content matches style options
- Optionally, update the block to change the style options, save the block, and check clicking pager links again

### Unique ID feature for blocks
- Add a unique ID to the block you were just editing
- Save the block, save the layout
- Confirm that the ID is rendered in the HTML markup for the block
- Edit the layout, edit the block
- Change the unique ID, save the block, save the layout
- Confirm that the new ID is rendered in the HTML markup for the block

### Unpublished node in menu selection
- https://sandbox.uiowa.ddev.site/admin/structure/menu/manage/main/add
- In the "Link" field, start typing "Link for" and confirm that "Link for unpublished auto-complete" shows up as a suggestion and can be selected.

### Able to access the LB entity from `'#entity'` when in the LB context
- The best way I could think of to test this is to set a breakpoint here: https://github.com/uiowa/uiowa/blob/6c61debb1e89f57bf8301d25e0f3738e11b2a12d/docroot/themes/custom/uids_base/uids_base.theme#L515 and then ensure that the `$node` object is populated when visiting the Layout tab on a page.
